my-app/src/reducers/index.js

```diff
@@ -1,17 +1,50 @@
 // add a reducer file and build out a reducer with a default return. Build your initial State object and export both the initial State and reducer.
-export const initialState ={
-    item: 'Learn about reducers',
-    completed: false,
-    id: 3389820
-};
+export const initialState = {
+    listItem: '',
+    todoList: []
```

Comment:
Hi Miley I was looking over your reducer code. Just a few tips. You can create a unique ID for your todo list and add this id to State. You can also setup for Error Checkin and create error on state like. 
`const intialState = {

    todo: [],
    loading: false,
   completed: false,
    error: ''
}`